### PR TITLE
fix/test(twap): remove v2 query proto wrap, undeprecate v1, test v1 twap API

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #2962

## What is the purpose of the change

We added v2 twap query proto wrap that misplaced parameters. This PR removes the faulty v2.

Additionally, it covers the original queries with tests to ensure that parameters are correct now.

Lastly, although `wasmbindings` are deprecated, they were also broken so this PR fixes them.

### CODE REVIEW

File: x/twap/client/query_proto_wrap_test.go [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+func TestKeeperTestSuite(t *testing.T) {
+	suite.Run(t, new(QueryTestSuite))
+}
+
+func (suite *QueryTestSuite) SetupTest() {
+	suite.Setup()
+}
+
+func (suite *QueryTestSuite) TestV2QueryTwap() {
```
Comment (p0mvn) [inline x/twap/client/query_proto_wrap_test.go:31]:
```suggestion
func (suite *QueryTestSuite) TestQueryTwap() {
```

File: x/twap/client/query_proto_wrap.go [hunk 2]
```diff
@@ -18,7 +18,10 @@ type Querier struct {
 func (q Querier) ArithmeticTwap(ctx sdk.Context,
 	req queryproto.ArithmeticTwapRequest, // nolint: staticcheck
 ) (*queryproto.ArithmeticTwapResponse, error) {
-	if (req.EndTime == nil || *req.EndTime == time.Time{}) {
+	if req.EndTime == nil {
+		req.EndTime = &time.Time{}
+	}
+	if (*req.EndTime == time.Time{}) {
```
Comment (ValarDragon) [inline x/twap/client/query_proto_wrap.go:24]: Why did this get changed? Old behavior looks correct to me

**Review by ValarDragon — APPROVED** [review]
This LGTM, as long as were very sure Twap v1 got it right

File: x/twap/client/query_proto_wrap.go (hunk 2 above)
Comment (p0mvn) [reply x/twap/client/query_proto_wrap.go:24]: if `req.EndTime` is nil, the old behavior would try to dereference a nil pointer.

**Pull request merged**